functions

```diff
@@ -14,7 +14,7 @@ service_connect() {
   local SERVICE="$1"
   local SERVICE_ROOT="$PLUGIN_DATA_ROOT/$SERVICE"
   local SERVICE_NAME="$(get_service_name "$SERVICE")"
-  local PASSWORD="$(cat "$SERVICE_ROOT/PASSWORD")"
+  local PASSWORD="$(cat "$SERVICE_ROOT/auth/mariadb")"
```

Comment:
you might want to turn `mariadb` into `$SERVICE` (lowercase)

Reply:
oh.. I might be wrong. what is service here?

Reply:
`$SERVICE` is a name of service, you create via 

```bash
dokku mariadb:create <service>
```